src/shared/components/plots/MultipleCategoryBarPlotUtils.ts

```diff
@@ -107,7 +116,8 @@ export function makeBarSpecs(
     categoryCoord: (categoryIndex: number) => number,
     horizontalBars: boolean,
     stacked: boolean,
-    percentage: boolean
+    percentage: boolean,
+    sortOption: string
```

Comment:
we can also make this optional so we don't have to pass `sortByAlphabet` every time we use this. and if nothing is passed, assume sortOption is using `sortByAlphabet`